traffic_router/core/src/main/scripts/postinstall.sh

```diff
@@ -27,6 +28,17 @@ if [ -f /opt/traffic_router/conf/*.crt ]; then
 fi
 
 
+echo -e "
+cd /opt/traffic_router/conf
+
+keytool=\$(dirname \$(readlink -f \$(which java)))/keytool
+
+if [ ! -f /opt/traffic_router/conf/keyStore.jks ]; then \n
+    \$keytool -genkeypair -v -alias \$(hostname -f) -dname \"CN=\$(hostname -f), OU=APIDefault, O=Apache Traffic Control, L=Denver, ST=Colorado, C=US\" -keystore \$(pwd)/keyStore.jks -storepass changeit -keyalg RSA -ext KeyUsage=\"digitalSignature,keyEncipherment,keyCertSign\" -ext BasicConstraints:\"critical=ca:true\" -storetype JKS
+    \$keytool -exportcert -v -alias \$(hostname -f) -file \$(hostname -f).crt -keypass changeit -storepass changeit -keystore \$(pwd)/keyStore.jks -rfc
```

Comment:
I had another thought when looking at the cert this generates. It appears by default a cert is only valid for about 90 days? Will that cause problems if TR isn't restarted within 90 days? Are we able to make it something crazy like 10 years if we want?

Reply:
done